src/miner.cpp

```diff
@@ -610,14 +611,16 @@ void BitcoinMiner(CWallet* pwallet, bool fProofOfStake)
                     continue;
             }
 
-            if (mapHashedBlocks.count(chainActive.Tip()->nHeight)) //search our map of hashed blocks, see if bestblock has been hashed yet
+            if (nHeightLast == chainActive.Tip()->nHeight)
```

Comment:
I don't think this accomplishes anything differently than how the code was before.

Reply:
I changed it to a code that I can understand.
If it is the same, there is no problem.